src/resources/ha-style.ts

```diff
@@ -201,6 +200,214 @@ documentContainer.innerHTML = `<custom-style>
       --material-caption-font-size: 0.75rem;
       --material-button-font-size: 0.875rem;
 
+      /* Paper shadow */
```

Comment:
Not sure we actually need these styles here, but this is what was previously imported with `@polymer/paper-styles/paper-styles` minus the bloat of Polymer...